src/test/mina_automation/executor.ml

```diff
@@ -22,14 +22,12 @@ end
 let logger = Logger.create ()
 
 module Make_PathFinder (P : AppPaths) = struct
-  let app_name_under_dune = P.dune_name
```

Comment:
Haha, was gonna ask what's the point of this alias, and then in this you deleted it :)